components/director/pkg/log/gql_ops_logger_test.go

```diff
@@ -0,0 +1,41 @@
+package log
+
+import (
+	"context"
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/vektah/gqlparser/v2/ast"
+	"testing"
+)
+
+const testOperationName = "myOperation"
+
+func TestGraphQlRequestDetailsLogging(t *testing.T) {
+	t.Run("log GQL query details", func(t *testing.T) {
+		ctx := ContextWithMdc(context.Background())
+		ctx = graphql.WithOperationContext(ctx, &graphql.OperationContext{
+			Operation: &ast.OperationDefinition{
+				Operation: ast.Query,
+				SelectionSet: []ast.Selection{&ast.Field{
+					Name: testOperationName,
+				}},
+			},
+		})
+
+		middleware := NewGqlLoggingInterceptor()
+		middleware.InterceptOperation(ctx, func(ctx context.Context) graphql.ResponseHandler {
+			return nil
+		})
+
+		mdc := MdcFromContext(ctx)
+		opType, ok := mdc.mdc[logKeyOperationType]
```

Comment:
For consistency with all other unit tests I'd use the require/assert package instead of checking for string equality and then using t.Errorf

Reply:
fixed